kiota_http/middleware/redirect_handler.py

```diff
@@ -174,7 +174,7 @@ def _redirect_url(
         except Exception as exc:
             raise Exception(f"Invalid URL in location header: {exc}.")
 
-        if url.scheme != request.url.scheme and not options.allow_redirect_on_scheme_change:
+        if not u.is_relative_url and url.scheme != request.url.scheme and not options.allow_redirect_on_scheme_change:
```

Comment:
```suggestion
        if not url.is_relative_url and url.scheme != request.url.scheme and not options.allow_redirect_on_scheme_change:
```

Should this be `url`

Reply:
wops sorry